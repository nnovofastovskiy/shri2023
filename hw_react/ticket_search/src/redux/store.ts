import { configureStore, getDefaultMiddleware } from "@reduxjs/toolkit";
import { cartReducer } from "./features/cart";
import { genresReducer } from "./features/genres";
import { filterReducer } from "./features/filter";
import { movieApi } from "./services/movieApi";

export const store = configureStore({
    reducer: {
        cart: cartReducer,
        genres: genresReducer,
        filter: filterReducer,
        [movieApi.reducerPath]: movieApi.reducer
    },
    middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(movieApi.middleware),
    devTools: process.env.NODE_ENV !== 'production'
})