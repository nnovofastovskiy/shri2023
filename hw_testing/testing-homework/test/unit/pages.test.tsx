import React from 'react';
import { it, describe, expect } from '@jest/globals';

import { render, within, getByText, screen } from '@testing-library/react';

import { BrowserRouter, MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import { Application } from '../../src/client/Application'
import { initStore } from '../../src/client/store';
import { ExampleApi, CartApi } from '../../src/client/api';
import { data } from './data';

describe('в магазине должны быть страницы:', () => {


    it('главная', () => {
        const basename = data.basename;
        const api = new ExampleApi(basename);
        const cart = new CartApi();
        const store = initStore(api, cart);
        const application = (
            <MemoryRouter initialEntries={[data.pages.home]}>
                <Provider store={store}>
                    <Application />
                </Provider>
            </MemoryRouter>
        );
        const homePage = render(application);
        expect(homePage).toMatchSnapshot();
    });
    it('каталог', () => {
        const basename = data.basename;
        const api = new ExampleApi(basename);
        const cart = new CartApi();
        const store = initStore(api, cart);
        const application = (
            <MemoryRouter initialEntries={[data.pages.catalog]}>
                <Provider store={store}>
                    <Application />
                </Provider>
            </MemoryRouter>
        );
        const homePage = render(application);
        expect(homePage).toMatchSnapshot();
    });
    it('условия доставки', () => {
        const basename = data.basename;
        const api = new ExampleApi(basename);
        const cart = new CartApi();
        const store = initStore(api, cart);
        const application = (
            <MemoryRouter initialEntries={[data.pages.delivery]}>
                <Provider store={store}>
                    <Application />
                </Provider>
            </MemoryRouter>
        );
        const homePage = render(application);
        expect(homePage).toMatchSnapshot();
    });
    it('контакты', () => {
        const basename = data.basename;
        const api = new ExampleApi(basename);
        const cart = new CartApi();
        const store = initStore(api, cart);
        const application = (
            <MemoryRouter initialEntries={[data.pages.contacts]}>
                <Provider store={store}>
                    <Application />
                </Provider>
            </MemoryRouter>
        );
        const homePage = render(application);
        expect(homePage).toMatchSnapshot();
    });
});
