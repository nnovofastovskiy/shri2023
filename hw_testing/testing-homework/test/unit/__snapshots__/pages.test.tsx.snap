// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`в магазине должны быть страницы: главная 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Application"
      >
        <nav
          class="navbar navbar-expand-sm navbar-light bg-light"
        >
          <div
            class="container"
          >
            <a
              class="Application-Brand navbar-brand"
              href="/"
            >
              Example store
            </a>
            <button
              aria-label="Toggle navigation"
              class="Application-Toggler navbar-toggler"
            >
              <span
                class="navbar-toggler-icon"
              />
            </button>
            <div
              class="Application-Menu collapse navbar-collapse"
            >
              <div
                class="navbar-nav"
              >
                <a
                  class="nav-link"
                  href="/catalog"
                >
                  Catalog
                </a>
                <a
                  class="nav-link"
                  href="/delivery"
                >
                  Delivery
                </a>
                <a
                  class="nav-link"
                  href="/contacts"
                >
                  Contacts
                </a>
                <a
                  class="nav-link"
                  href="/cart"
                >
                  Cart
                </a>
              </div>
            </div>
          </div>
        </nav>
        <div
          class="container pt-4"
        >
          <div
            class="Home"
          >
            <div
              class="row"
            >
              <div
                class="col bg-secondary text-white py-4 bg-opacity-75"
              >
                <p
                  class="display-3"
                >
                  Welcome to Example store!
                </p>
                <p
                  class="lead"
                >
                  Culpa perspiciatis corporis facilis fugit similique
                </p>
                <p
                  class="lead"
                >
                  Cum aut ut eveniet rem cupiditate natus veritatis quia
                </p>
              </div>
            </div>
            <div
              class="row mb-4"
            >
              <div
                class="col-12 col-md-4 bg-light py-3"
              >
                <h1>
                  Quickly
                </h1>
                <p
                  class="lead"
                >
                  Odio aut assumenda ipsam amet reprehenderit. Perspiciatis qui molestiae qui tempora quisquam
                </p>
              </div>
              <div
                class="col-12 col-md-4 bg-light py-3"
              >
                <h1>
                  Qualitatively
                </h1>
                <p
                  class="lead"
                >
                  Ut nisi distinctio est non voluptatem. Odio aut assumenda ipsam amet reprehenderit
                </p>
              </div>
              <div
                class="col-12 col-md-4 bg-light py-3"
              >
                <h1>
                  Inexpensive
                </h1>
                <p
                  class="lead"
                >
                  Perspiciatis qui molestiae qui tempora quisquam. Ut nisi distinctio est non voluptatem
                </p>
              </div>
            </div>
            <div
              class="row mb-4"
            >
              <div
                class="col-12py-3"
              >
                <p>
                  Sed voluptatum quis voluptates laudantium incidunt laudantium. Illo non quos eos vel ipsa. Explicabo itaque est optio neque rerum provident enim qui sed. Corrupti commodi voluptatem vero soluta hic.
                </p>
                <p>
                  Modi corporis consectetur aliquid sit cum tenetur enim. Sed voluptatum quis voluptates laudantium incidunt laudantium. Illo non quos eos vel ipsa. Explicabo itaque est optio neque rerum provident enim qui sed. Corrupti commodi voluptatem vero soluta hic.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Application"
    >
      <nav
        class="navbar navbar-expand-sm navbar-light bg-light"
      >
        <div
          class="container"
        >
          <a
            class="Application-Brand navbar-brand"
            href="/"
          >
            Example store
          </a>
          <button
            aria-label="Toggle navigation"
            class="Application-Toggler navbar-toggler"
          >
            <span
              class="navbar-toggler-icon"
            />
          </button>
          <div
            class="Application-Menu collapse navbar-collapse"
          >
            <div
              class="navbar-nav"
            >
              <a
                class="nav-link"
                href="/catalog"
              >
                Catalog
              </a>
              <a
                class="nav-link"
                href="/delivery"
              >
                Delivery
              </a>
              <a
                class="nav-link"
                href="/contacts"
              >
                Contacts
              </a>
              <a
                class="nav-link"
                href="/cart"
              >
                Cart
              </a>
            </div>
          </div>
        </div>
      </nav>
      <div
        class="container pt-4"
      >
        <div
          class="Home"
        >
          <div
            class="row"
          >
            <div
              class="col bg-secondary text-white py-4 bg-opacity-75"
            >
              <p
                class="display-3"
              >
                Welcome to Example store!
              </p>
              <p
                class="lead"
              >
                Culpa perspiciatis corporis facilis fugit similique
              </p>
              <p
                class="lead"
              >
                Cum aut ut eveniet rem cupiditate natus veritatis quia
              </p>
            </div>
          </div>
          <div
            class="row mb-4"
          >
            <div
              class="col-12 col-md-4 bg-light py-3"
            >
              <h1>
                Quickly
              </h1>
              <p
                class="lead"
              >
                Odio aut assumenda ipsam amet reprehenderit. Perspiciatis qui molestiae qui tempora quisquam
              </p>
            </div>
            <div
              class="col-12 col-md-4 bg-light py-3"
            >
              <h1>
                Qualitatively
              </h1>
              <p
                class="lead"
              >
                Ut nisi distinctio est non voluptatem. Odio aut assumenda ipsam amet reprehenderit
              </p>
            </div>
            <div
              class="col-12 col-md-4 bg-light py-3"
            >
              <h1>
                Inexpensive
              </h1>
              <p
                class="lead"
              >
                Perspiciatis qui molestiae qui tempora quisquam. Ut nisi distinctio est non voluptatem
              </p>
            </div>
          </div>
          <div
            class="row mb-4"
          >
            <div
              class="col-12py-3"
            >
              <p>
                Sed voluptatum quis voluptates laudantium incidunt laudantium. Illo non quos eos vel ipsa. Explicabo itaque est optio neque rerum provident enim qui sed. Corrupti commodi voluptatem vero soluta hic.
              </p>
              <p>
                Modi corporis consectetur aliquid sit cum tenetur enim. Sed voluptatum quis voluptates laudantium incidunt laudantium. Illo non quos eos vel ipsa. Explicabo itaque est optio neque rerum provident enim qui sed. Corrupti commodi voluptatem vero soluta hic.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`в магазине должны быть страницы: каталог 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Application"
      >
        <nav
          class="navbar navbar-expand-sm navbar-light bg-light"
        >
          <div
            class="container"
          >
            <a
              class="Application-Brand navbar-brand"
              href="/"
            >
              Example store
            </a>
            <button
              aria-label="Toggle navigation"
              class="Application-Toggler navbar-toggler"
            >
              <span
                class="navbar-toggler-icon"
              />
            </button>
            <div
              class="Application-Menu collapse navbar-collapse"
            >
              <div
                class="navbar-nav"
              >
                <a
                  aria-current="page"
                  class="nav-link active"
                  href="/catalog"
                >
                  Catalog
                </a>
                <a
                  class="nav-link"
                  href="/delivery"
                >
                  Delivery
                </a>
                <a
                  class="nav-link"
                  href="/contacts"
                >
                  Contacts
                </a>
                <a
                  class="nav-link"
                  href="/cart"
                >
                  Cart
                </a>
              </div>
            </div>
          </div>
        </nav>
        <div
          class="container pt-4"
        >
          <div
            class="Catalog"
          >
            <div
              class="row"
            >
              <div
                class="col"
              >
                <h1>
                  Catalog
                </h1>
              </div>
            </div>
            <div
              class="row"
            >
              LOADING
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Application"
    >
      <nav
        class="navbar navbar-expand-sm navbar-light bg-light"
      >
        <div
          class="container"
        >
          <a
            class="Application-Brand navbar-brand"
            href="/"
          >
            Example store
          </a>
          <button
            aria-label="Toggle navigation"
            class="Application-Toggler navbar-toggler"
          >
            <span
              class="navbar-toggler-icon"
            />
          </button>
          <div
            class="Application-Menu collapse navbar-collapse"
          >
            <div
              class="navbar-nav"
            >
              <a
                aria-current="page"
                class="nav-link active"
                href="/catalog"
              >
                Catalog
              </a>
              <a
                class="nav-link"
                href="/delivery"
              >
                Delivery
              </a>
              <a
                class="nav-link"
                href="/contacts"
              >
                Contacts
              </a>
              <a
                class="nav-link"
                href="/cart"
              >
                Cart
              </a>
            </div>
          </div>
        </div>
      </nav>
      <div
        class="container pt-4"
      >
        <div
          class="Catalog"
        >
          <div
            class="row"
          >
            <div
              class="col"
            >
              <h1>
                Catalog
              </h1>
            </div>
          </div>
          <div
            class="row"
          >
            LOADING
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`в магазине должны быть страницы: контакты 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Application"
      >
        <nav
          class="navbar navbar-expand-sm navbar-light bg-light"
        >
          <div
            class="container"
          >
            <a
              class="Application-Brand navbar-brand"
              href="/"
            >
              Example store
            </a>
            <button
              aria-label="Toggle navigation"
              class="Application-Toggler navbar-toggler"
            >
              <span
                class="navbar-toggler-icon"
              />
            </button>
            <div
              class="Application-Menu collapse navbar-collapse"
            >
              <div
                class="navbar-nav"
              >
                <a
                  class="nav-link"
                  href="/catalog"
                >
                  Catalog
                </a>
                <a
                  class="nav-link"
                  href="/delivery"
                >
                  Delivery
                </a>
                <a
                  aria-current="page"
                  class="nav-link active"
                  href="/contacts"
                >
                  Contacts
                </a>
                <a
                  class="nav-link"
                  href="/cart"
                >
                  Cart
                </a>
              </div>
            </div>
          </div>
        </nav>
        <div
          class="container pt-4"
        >
          <div
            class="Contacts"
          >
            <div
              class="row"
            >
              <div
                class="col"
              >
                <h1>
                  Contacts
                </h1>
                <p>
                  Ut non consequatur aperiam ex dolores. Voluptatum harum consequatur est totam. Aut voluptatum aliquid aut optio et ea. Quaerat et eligendi minus quasi. Culpa voluptatem voluptatem dolores molestiae aut quos iure. Repellat aperiam ut aliquam iure. Veritatis magnam quisquam et dolorum recusandae aut.
                </p>
                <p>
                  Molestias inventore illum architecto placeat molestias ipsam facilis ab quo. Rem dolore cum qui est reprehenderit assumenda voluptatem nisi ipsa. Unde libero quidem. Excepturi maiores vel quia. Neque facilis nobis minus veniam id. Eum cum eveniet accusantium molestias voluptas aut totam laborum aut. Ea molestiae ullam et. Quis ea ipsa culpa eligendi ab sit ea error suscipit. Quia ea ut minus distinctio quam eveniet nihil. Aut voluptate numquam ipsa dolorem et quas nemo.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Application"
    >
      <nav
        class="navbar navbar-expand-sm navbar-light bg-light"
      >
        <div
          class="container"
        >
          <a
            class="Application-Brand navbar-brand"
            href="/"
          >
            Example store
          </a>
          <button
            aria-label="Toggle navigation"
            class="Application-Toggler navbar-toggler"
          >
            <span
              class="navbar-toggler-icon"
            />
          </button>
          <div
            class="Application-Menu collapse navbar-collapse"
          >
            <div
              class="navbar-nav"
            >
              <a
                class="nav-link"
                href="/catalog"
              >
                Catalog
              </a>
              <a
                class="nav-link"
                href="/delivery"
              >
                Delivery
              </a>
              <a
                aria-current="page"
                class="nav-link active"
                href="/contacts"
              >
                Contacts
              </a>
              <a
                class="nav-link"
                href="/cart"
              >
                Cart
              </a>
            </div>
          </div>
        </div>
      </nav>
      <div
        class="container pt-4"
      >
        <div
          class="Contacts"
        >
          <div
            class="row"
          >
            <div
              class="col"
            >
              <h1>
                Contacts
              </h1>
              <p>
                Ut non consequatur aperiam ex dolores. Voluptatum harum consequatur est totam. Aut voluptatum aliquid aut optio et ea. Quaerat et eligendi minus quasi. Culpa voluptatem voluptatem dolores molestiae aut quos iure. Repellat aperiam ut aliquam iure. Veritatis magnam quisquam et dolorum recusandae aut.
              </p>
              <p>
                Molestias inventore illum architecto placeat molestias ipsam facilis ab quo. Rem dolore cum qui est reprehenderit assumenda voluptatem nisi ipsa. Unde libero quidem. Excepturi maiores vel quia. Neque facilis nobis minus veniam id. Eum cum eveniet accusantium molestias voluptas aut totam laborum aut. Ea molestiae ullam et. Quis ea ipsa culpa eligendi ab sit ea error suscipit. Quia ea ut minus distinctio quam eveniet nihil. Aut voluptate numquam ipsa dolorem et quas nemo.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`в магазине должны быть страницы: условия доставки 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Application"
      >
        <nav
          class="navbar navbar-expand-sm navbar-light bg-light"
        >
          <div
            class="container"
          >
            <a
              class="Application-Brand navbar-brand"
              href="/"
            >
              Example store
            </a>
            <button
              aria-label="Toggle navigation"
              class="Application-Toggler navbar-toggler"
            >
              <span
                class="navbar-toggler-icon"
              />
            </button>
            <div
              class="Application-Menu collapse navbar-collapse"
            >
              <div
                class="navbar-nav"
              >
                <a
                  class="nav-link"
                  href="/catalog"
                >
                  Catalog
                </a>
                <a
                  aria-current="page"
                  class="nav-link active"
                  href="/delivery"
                >
                  Delivery
                </a>
                <a
                  class="nav-link"
                  href="/contacts"
                >
                  Contacts
                </a>
                <a
                  class="nav-link"
                  href="/cart"
                >
                  Cart
                </a>
              </div>
            </div>
          </div>
        </nav>
        <div
          class="container pt-4"
        >
          <div
            class="Delivery"
          >
            <div
              class="row"
            >
              <div
                class="col"
              >
                <h1>
                  Delivery
                </h1>
                <p>
                  Deserunt occaecati tempora. Qui occaecati est aliquam. Enim qui nulla ipsam. Incidunt impedit enim consequuntur amet at consequuntur vero. Dolor et ad facere asperiores iste est praesentium quaerat iure. Quibusdam mollitia autem quos voluptas quia est doloremque corporis et. Sed fuga quasi esse perspiciatis fugit maxime. Qui quidem amet.
                </p>
                <img
                  class="Image w-25 mb-4"
                  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkMAYAADkANVKH3ScAAAAASUVORK5CYII="
                />
                <p>
                  Dolores magnam consequatur iste aliquam qui sint non ab. Culpa saepe omnis. Recusandae vel aperiam voluptates harum. Perspiciatis qui molestiae qui tempora quisquam. Mollitia voluptatum minus laboriosam. Dolor maiores possimus repudiandae praesentium hic eos. Veritatis et repellat.
                </p>
                <p>
                  Pariatur nisi nobis hic ut facilis sunt rerum id error. Soluta nihil quisquam quia rerum illo. Ipsam et suscipit est iure incidunt quasi et eum. Culpa libero dignissimos recusandae. In magni sapiente non voluptas molestias. Deserunt quos quo placeat sunt. Ea necessitatibus dolores eaque ex aperiam sunt eius. Saepe aperiam aut. Quaerat natus consequatur aut est id saepe et aut facilis.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Application"
    >
      <nav
        class="navbar navbar-expand-sm navbar-light bg-light"
      >
        <div
          class="container"
        >
          <a
            class="Application-Brand navbar-brand"
            href="/"
          >
            Example store
          </a>
          <button
            aria-label="Toggle navigation"
            class="Application-Toggler navbar-toggler"
          >
            <span
              class="navbar-toggler-icon"
            />
          </button>
          <div
            class="Application-Menu collapse navbar-collapse"
          >
            <div
              class="navbar-nav"
            >
              <a
                class="nav-link"
                href="/catalog"
              >
                Catalog
              </a>
              <a
                aria-current="page"
                class="nav-link active"
                href="/delivery"
              >
                Delivery
              </a>
              <a
                class="nav-link"
                href="/contacts"
              >
                Contacts
              </a>
              <a
                class="nav-link"
                href="/cart"
              >
                Cart
              </a>
            </div>
          </div>
        </div>
      </nav>
      <div
        class="container pt-4"
      >
        <div
          class="Delivery"
        >
          <div
            class="row"
          >
            <div
              class="col"
            >
              <h1>
                Delivery
              </h1>
              <p>
                Deserunt occaecati tempora. Qui occaecati est aliquam. Enim qui nulla ipsam. Incidunt impedit enim consequuntur amet at consequuntur vero. Dolor et ad facere asperiores iste est praesentium quaerat iure. Quibusdam mollitia autem quos voluptas quia est doloremque corporis et. Sed fuga quasi esse perspiciatis fugit maxime. Qui quidem amet.
              </p>
              <img
                class="Image w-25 mb-4"
                src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkMAYAADkANVKH3ScAAAAASUVORK5CYII="
              />
              <p>
                Dolores magnam consequatur iste aliquam qui sint non ab. Culpa saepe omnis. Recusandae vel aperiam voluptates harum. Perspiciatis qui molestiae qui tempora quisquam. Mollitia voluptatum minus laboriosam. Dolor maiores possimus repudiandae praesentium hic eos. Veritatis et repellat.
              </p>
              <p>
                Pariatur nisi nobis hic ut facilis sunt rerum id error. Soluta nihil quisquam quia rerum illo. Ipsam et suscipit est iure incidunt quasi et eum. Culpa libero dignissimos recusandae. In magni sapiente non voluptas molestias. Deserunt quos quo placeat sunt. Ea necessitatibus dolores eaque ex aperiam sunt eius. Saepe aperiam aut. Quaerat natus consequatur aut est id saepe et aut facilis.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
